File: back/src/main/kotlin/br/ufsc/mariaeduardamelohang/servidormatriculaufscpwa/model/DiaSemana.java
package br.ufsc.mariaeduardamelohang.servidormatriculaufscpwa.model;

import static javax.persistence.GenerationType.IDENTITY;

import java.io.Serializable;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "TB_DIA_SEMANA")
public class DiaSemana implements Serializable {

	private static final long serialVersionUID = -3453802262703116799L;

	@Id
	@GeneratedValue(strategy = IDENTITY)
	Long id;

	@Column(nullable = false)
	String nome;
}

File: back/src/main/kotlin/br/ufsc/mariaeduardamelohang/servidormatriculaufscpwa/model/DiaSemana.kt
package br.ufsc.mariaeduardamelohang.servidormatriculaufscpwa.model

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType.IDENTITY
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "TB_DIA_SEMANA")
data class DiaSemana(
    @Id
    @GeneratedValue(strategy = IDENTITY)
    var id: Long? = null,

    @Column(nullable = false)
    val nome: String
)